src/script/view_model/bindings/CommonBindings.ts

```diff
@@ -326,15 +329,15 @@ ko.bindingHandlers.load_image_on_hover = {
     const animated_gif = hoverable_item.data('hover');
 
     if (animated_gif) {
-      let image = undefined;
+      let image: HTMLImageElement = undefined;
       hoverable_item
-        .on('mouseover', function () {
+        .on('mouseover', () => {
           const item = $(this);
```

Comment:
We need to keep `function ()` here because two in the next line we are using `this`.